pages/api/extension/trigger.ts

```diff
@@ -105,4 +105,12 @@ function parseURL(url: string) {
     };
 }
 
-
+function prepareBody(repoProvider: string, repoOwner: string, repoName: string, prNumber: string, repoConfig: any) {
+    return {
+        repo_provider: repoProvider,
+        repo_owner: repoOwner,
+        repo_name: repoName,
+        pr_number: prNumber,
+        repo_config: repoConfig,
+    }
+}
```

Comment:
Any reason why you decided to turn this single line assignment into a function call?

Reply:
no reason, fixing